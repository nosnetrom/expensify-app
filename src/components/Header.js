import React from 'react';
import { NavLink } from 'react-router-dom';

const Header = () => (
  <header>
    <h1>Expensify</h1>
    <p>
      <NavLink to="/" exact={true} activeClassName="isActive">Dashboard</NavLink>&nbsp;
      <NavLink to="/create" activeClassName="isActive">Create Expense</NavLink>&nbsp;
      <NavLink to="/help" activeClassName="isActive">Help</NavLink>
    </p>
  </header>
);

export default Header;