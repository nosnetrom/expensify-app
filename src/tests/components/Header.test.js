import React from 'react';
import { shallow } from 'enzyme';
import Header from '../../components/Header';

test('test to render Header', () => {
  const wrapper = shallow(<Header />);
  expect(wrapper).toMatchSnapshot();
});

