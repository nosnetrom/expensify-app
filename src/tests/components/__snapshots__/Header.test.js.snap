// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test to render Header 1`] = `
<header>
  <h1>
    Expensify
  </h1>
  <p>
    <NavLink
      activeClassName="isActive"
      ariaCurrent="true"
      exact={true}
      to="/"
    >
      Dashboard
    </NavLink>
     
    <NavLink
      activeClassName="isActive"
      ariaCurrent="true"
      to="/create"
    >
      Create Expense
    </NavLink>
     
    <NavLink
      activeClassName="isActive"
      ariaCurrent="true"
      to="/help"
    >
      Help
    </NavLink>
  </p>
</header>
`;
